server/routes/authorization.js

```diff
@@ -4,10 +4,10 @@ import config from '../lib/config';
 import * as authorization from '../lib/authorization';
 import { requireScope } from '../lib/middlewares';
 
-export default (storage) => {
+export default (auth0, storage) => {
```

Comment:
Follows the same patter as in `applications`, where the middleware is passed as parameter.